src/main/java/com/google/sps/servlets/DirectionsServlet.java

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.repackaged.com.google.gson.Gson;
+import com.google.sps.utility.DirectionsUtility;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Arrays;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+@WebServlet("/directions")
+public class DirectionsServlet extends HttpServlet {
```

Comment:
Public classes require javadoc. [7.3](https://google.github.io/styleguide/javaguide.html#s7.1-javadoc-formatting)

Reply:
I have incorporated this in the new dependency injection ready PR #63 . Thanks!